cmd/podman/main.go

```diff
@@ -40,6 +41,11 @@ func main() {
 		return
 	}
 
+	cgroupv2, _ := cgroups.IsCgroup2UnifiedMode()
+	if !cgroupv2 {
+		logrus.Warnf("WARNING! You are using cgroups-v1 which is not supported with Podman v5 and will be removed in a future version. Consider switching to cgroups-v2.")
```

Comment:
nit, `Warnf` alread prefixes the log level. We can also use `Warn` without the format specifier:

```suggestion
		logrus.Warn("Using cgroups-v1 which is not supported with Podman v5 and will be removed in a future version. Consider switching to cgroups-v2.")
```

Reply:
I'll add this for now. Thanks.